common/scala/Dockerfile

```diff
@@ -1,8 +1,5 @@
-FROM buildpack-deps:trusty-curl
+FROM alpine:3.6
```

Comment:
WDYT about extending from an existing JAVA based image from https://hub.docker.com/_/openjdk/ ?

Reply:
switching jdk from oracle to openjdk should be a separate topic, I think; I'm not sure all the ramifications there, I believe @bwmcadams ran into some issue with openjdk in recent times?

Reply:
I tried OpenJDK once and it was fine at that time. FWIW you're right though, that should be an entirely seperate discussion.

Reply:
Agreed there needs to be a conversation around JDK.  Oracle doesn't support the s390x architecture I'm working on; I am working with openjdk there but haven't completed regression testing.